Xamarin.Forms.Maps/Pin.cs

```diff
@@ -2,17 +2,15 @@
 
 namespace Xamarin.Forms.Maps
 {
-	public sealed class Pin : BindableObject
+	public class Pin : BindableObject
```

Comment:
For the moment I think I'm okay with unsealing this class. We will need to evaluate the other renderers to make sure there isn't a reason this could go wrong.